flask_app/backend_jobs/ingestion_pipeline/main.py

```diff
@@ -55,13 +55,17 @@ def _is_valid_image(image):
     """ This function returns whether the given image satisfies minimum requirements of the platform
     e.g. url != none
```

Comment:
update the docstring

Reply:
switched to -
This function returns whether the given image contains a valid url.